device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py

```diff
@@ -212,7 +212,7 @@ def get_transceiver_change_event(self, timeout=0):
             from swsscommon import swsscommon
             self.state_db = swsscommon.DBConnector("STATE_DB",
                                                    REDIS_TIMEOUT_USECS,
-                                                   True)
```

Comment:
>True [](http://example.com/codeflow?start=51&length=4)

since this file is vendor owned. please have a smaller PR and ask vendor to review.

Reply:
Fixed